src/handlers/shortcut.rs

```diff
@@ -53,5 +64,26 @@ pub(super) async fn handle_command(
             .await?;
     }
 
+    // We add a small reminder for the author to the `@bot ready` when ready
+    if matches!(input, ShortcutCommand::Author) {
+        // Get the state of the author reminder for this PR
+        let mut db = ctx.db.get().await;
+        let mut state: IssueData<'_, AuthorReminderState> =
+            IssueData::load(&mut db, &issue, AUTHOR_REMINDER_KEY).await?;
+
+        if state.data.reminder_comment.is_none() {
+            let comment_body = format!(
+                "Reminder, use `@{bot} ready` to indicate that the PR is ready for review.",
```

Comment:
This makes it sound like they should execute the command right away, IMO. I would change the text to something like "Reminder, once the PR will become ready for a review, use `@{bot} ready`".

Reply:
Sure, changed to your suggested text.